specification/billing/resource-manager/Microsoft.Billing/stable/2023-04-01/transaction.json

```diff
@@ -63,259 +562,377 @@
         },
         "customerId": {
-          "description": "The ID of the customer for which the transaction took place. The field is applicable only for Microsoft Partner Agreement billing account.",
-          "readOnly": true,
+          "description": "The fully qualified ID that uniquely identifies a customer.",
           "type": "string"
         },
+        "creditType": {
+          "description": "The credit type of the transaction. Applies only to credited transactions.",
+          "enum": [
+            "Other",
+            "AzureFreeCredit",
+            "AzureCreditOffer",
+            "ServiceInterruption",
+            "Refund"
+          ],
+          "type": "string",
+          "x-ms-enum": {
+            "name": "CreditType",
+            "modelAsString": true
+          }
+        },
         "date": {
           "description": "The date of transaction.",
-          "format": "date-time",
-          "readOnly": true,
           "type": "string"
         },
         "discount": {
+          "format": "double",
           "description": "The percentage discount, if any, applied to this transaction.",
-          "readOnly": true,
           "type": "number"
         },
         "effectivePrice": {
-          "$ref": "./types.json#/definitions/Amount",
           "description": "The price of the product after applying any discounts.",
-          "readOnly": true
+          "allOf": [
+            {
+              "$ref": "./types.json#/definitions/Amount"
+            }
+          ]
         },
         "exchangeRate": {
+          "format": "double",
           "description": "The exchange rate used to convert charged amount to billing currency, if applicable.",
-          "readOnly": true,
           "type": "number"
         },
         "invoice": {
           "description": "Invoice on which the transaction was billed or 'pending' if the transaction is not billed.",
-          "readOnly": true,
           "type": "string"
         },
         "invoiceId": {
           "description": "The ID of the invoice on which the transaction was billed. This field is only applicable for transactions which are billed.",
-          "readOnly": true,
           "type": "string"
         },
         "invoiceSectionDisplayName": {
-          "description": "The name of the invoice section which will be billed for the transaction.",
-          "readOnly": true,
+          "description": "The name of the invoice section.",
           "type": "string"
         },
         "invoiceSectionId": {
-          "description": "The ID of the invoice section which will be billed for the transaction.",
-          "readOnly": true,
+          "description": "The fully qualified ID that uniquely identifies an invoice section.",
           "type": "string"
         },
+        "isThirdParty": {
+          "description": "Whether or not the transaction is third party.",
+          "type": "boolean"
+        },
         "kind": {
```

Comment:
Understand what this field comes from and why the description is different from the values

Reply:
It's a passthrough from MCAPI line item. We do no work on this property.